internal/provider/permission_data_source_test.go

```diff
@@ -1,12 +1,16 @@
 package provider
 
 import (
+	"os"
 	"testing"
 
 	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
 )
 
 func TestAccPermissionDataSource(t *testing.T) {
+	if os.Getenv("UNLEASH_ENTERPRISE") == "false" {
```

Comment:
Only noticed this now and this isn't a problem but the enterprise tests are opt out? This check would pass if the env var is not set?

Reply:
Spot on! It should be `os.Getenv("UNLEASH_ENTERPRISE") != "true"`

Reply:
[f2270a0](https://github.com/Unleash/terraform-provider-unleash/pull/58/commits/f2270a0e998ba1d8cad384311d790b55a6985eb5)